apps/user-portal/src/api/security-questions.ts

```diff
@@ -58,16 +58,16 @@ export const getSecurityQs = (): Promise<any> => {
             url: store.getState().config.endpoints.challengeAnswers
         };
 
-        return httpClient.request(requestConfig);
+        return httpClient(requestConfig);
     };
 
-    return httpClient.all([ getQuestions(), getAnswers() ])
-        .then(httpClient.spread((questions, answers) => {
+    return Promise.all([ getQuestions(), getAnswers() ])
```

Comment:
Promise all might need some pollyfills to work in IE environments. Can't we bind `all` and `spread` to the worker instance?

Reply:
I added a new method called `httpRequestAll` to imitate the `axios.all` method.